res/skins/QMLDemo/HotcueButton.qml

```diff
@@ -54,6 +52,9 @@ Skin.Button {
         anchors.fill: parent
         acceptedButtons: Qt.RightButton
         onClicked: {
+            if (hotcueStatusControl.value == 0)
```

Comment:
encapsulate as property on the CO? eg
```qml
readonly property bool enabled: this.value == 0
```

Reply:
I still don't like this. The meaning of special values should be encapsulated by the QML ControlProxy component and not bleed into other components...